.github/ISSUE_TEMPLATE/01-feature_request.md

```diff
@@ -11,10 +11,10 @@ labels: 'type: Feature Request'
 <!--
 Is your feature request related to an issue? Please describe.
 Describe the solution you'd like.
-Describe any work-arounds you've considered.
+Describe any work-arounds you've considered, if any.
```

Comment:
"any… if any" seems redundant to me.  I'd suggest either "Describe the workarounds you've tried, if any" or "Describe any workarounds you've explored."